build/archives/BUILD.gn

```diff
@@ -0,0 +1,64 @@
+import("//flutter/build/zip_bundle.gni")
+
+zip_bundle("artifacts") {
+  output = "$host_os-$target_cpu/artifacts.zip"
+  files = [
+    {
+      source = "//third_party/icu/flutter/icudtl.dat"
+      destination = "icudtl.dat"
+    },
+    {
+      source = "$root_out_dir/gen/flutter/lib/snapshot/isolate_snapshot.bin"
+      destination = "isolate_snapshot.bin"
+    },
+    {
+      source = "$root_gen_dir/flutter/lib/snapshot/vm_isolate_snapshot.bin"
+      destination = "vm_isolate_snapshot.bin"
+    },
+    {
+      source = "$root_gen_dir/frontend_server.dart.snapshot"
+      destination = "frontend_server.dart.snapshot"
+    },
+  ]
+  if (host_os == "win") {
+    files += [
+      {
+        source = "$root_out_dir/flutter_tester.exe"
+        destination = "flutter_tester"
+      },
+    ]
+  } else {
+    files += [
+      {
+        source = "$root_out_dir/flutter_tester"
+        destination = "flutter_tester"
+      },
+    ]
+  }
+  deps = [
```

Comment:
[optional] This is just a personal nit, but I find GN rules easier to read when the `deps` section is at the top.

Reply:
Deps moved to the top.